docs/DeveloperGuide.md

```diff
@@ -205,6 +205,55 @@ name, remark and product. Taking the aforementioned points into consideration, o
 implement the `find` command by taking in prefixes and throwing our relevant exceptions at appropriate points after
 considering code quality and end user experience.
 
+### Add supplier/warehouse feature
+The add supplier/warehouse mechanism is facilitated by the `AddCommandParser` and the `AddCommand`.
+The `AddCommandParser` implements `Parser` and the `AddCommand` extends `Command`, allowing the user to 
+add a supplier/warehouse to the app.
+
+#### What Add feature does
+The add feature allows users to add supplier/warehouse name, phone number and remarks. In addition, users
+can add email for suppliers and address for warehouses. Each supplier/warehouse is initialized without products associated to them.
+To add products, the addp feature should be invoked instead. Note that users are only able to add either
```

Comment:
It might be better to call the feature add product feature. 'addp' is the command word which should be used with the double tildas in my opinion

Reply:
This makes sense, I've edited it accordingly thanks!